vnext/local-cli/generator-windows/templates/cpp/proj/MyApp.vcxproj

```diff
@@ -151,16 +145,17 @@
       <DeploymentContent>false</DeploymentContent>
     </Text>
   </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="$(ReactNativeWindowsDir)\Microsoft.ReactNative\Microsoft.ReactNative.vcxproj">
-      <Project>{f7d32bd0-2749-483e-9a0d-1635ef7e3136}</Project>
-    </ProjectReference>
-  </ItemGroup>
-
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\Bundle.props" />
-  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\Bundle.Cpp.targets"/>
-
+  <ImportGroup Label="ReactNativeWindowsNodeTargets">
+    <Import Project="..\..\node_modules\react-native-windows\PropertySheets\External\Microsoft.ReactNative.CppApp.targets" Condition="Exists('..\..\node_modules\react-native-windows\PropertySheets\External\Microsoft.ReactNative.CppApp.targets')" />
+    <Import Project="AutolinkedNativeModules.g.targets" Condition="Exists('AutolinkedNativeModules.g.targets')" />
```

Comment:
can the autolinked native modules target just be imported in M.RN.targets?

Reply:
potentially we could have it reference the project folder root?

Reply:
We've had bugs before from bad assumptions about project layout, so reverse engineering user project directory layout from inside RNW gives me a bit of concern. I guess it might be okay using SolutionDir, but I can understand leaving this one in the template since it lives outside of RNW.

Reply:
yep, do-able.